telemetry/__fixtures__/build/index.html

```
@@ -0,0 +1 @@
+<html></html>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)